src/app/shared/shared.module.ts

```diff
@@ -64,32 +60,68 @@ import { NotificationPanelComponent } from './components/notification-panel/noti
 import { NotificationCardComponent } from './components/notification-card/notification-card.component'
 import { NotificationPopupContainerComponent } from './components/notification-popup-container/notification-popup-container.component'
 import { NotificationIconComponent } from './components/notification-icon/notification-icon.component'
+import { TranslateActionTypePipe } from './pipes/translate-action-type.pipe'
+import { CardComponent } from './components/card/card.component'
+import { TableComponent } from './components/table/table.component'
+import { ActionColorPipe } from './pipes/action-color.pipe'
+import { SidebarContainerComponent } from './components/sidebar-container/sidebar-container.component'
+import { TextInputComponent } from './components/text-input/text-input.component'
+import { NumberInputComponent } from './components/number-input/number-input.component'
+import { ButtonGroupComponent } from './components/button-group/button-group.component'
+import { MultiSelectComponent } from './components/multi-select/multi-select.component'
+import { DomFileDownloadService } from './services/dom-file-download.service'
+import { FileDownloadService } from './abstractions/file-download.service'
+import { JsonImportButtonComponent } from './components/json-import-button/json-import-button.component'
+import { JsonExportButtonComponent } from './components/json-export-button/json-export-button.component'
+import { CheckboxComponent } from './components/checkbox/checkbox.component'
+import { EmptyTableComponent } from './components/empty-table/empty-table.component'
+import { SelectComponent } from './components/select/select.component'
+import { KeyboardInputComponent } from './components/keyboard-input/keyboard-input.component'
+import { ActionBoxComponent } from './components/action-box/action-box.component'
+import { RoundedBoxComponent } from './components/rounded-box/rounded-box.component'
+import { RowComponent } from './components/row/row.component'
+import { ColumnComponent } from './components/column/column.component'
```

Comment:
Maybe it's not a bad idea to consider splitting the shared module into some smaller sub-modules.
Examples: shared.pipe.modules, shared.architecture.module, shared.components.module and ect.

Reply:
I have been contemplating the idea if we need a module for all our "basic scaffolding" components like buttons, tables, cards etc.

I didn't it yet since we need a clear distinction of what goes where. But it is definately a good topic for discussion :)